### PR TITLE
Add Workflow to Sync GitHub Issues to DevOps

### PR DESCRIPTION
**Pull request by kachawla**
Adding support for syncing GitHub issues to DevOps. Copied over from Radius repo: https://github.com/radius-project/radius/blob/main/.github/workflows/devops-boards.yaml

### CODE REVIEW

File: .github/workflows/~/workspace/latest/radius/.github/workflows/devops-boards.yaml
Comment (ytimocin) [inline .github/workflows/~/workspace/latest/radius/.github/workflows/devops-boards.yaml:1]: Is the yaml file in the correct folder? `.github/workflows/~/workspace/latest/radius/.github/workflows/devops-boards.yaml` is where it is supposed to be?

**Review by kachawla — COMMENTED** [review]
@sylvainsf @willtsai this is based on the decision made during triage. Please review.

**Review by sylvainsf — APPROVED** [review]
:shipit:

**Comment by sylvainsf** [comment]
First run failed with this issue, haven't had time to triage:
`Error: AADSTS700213: No matching federated identity record found for presented assertion subject 'repo:radius-project/community:environment:issues'. Please note that the matching is done using a case-sensitive comparison. Check your federated identity credential Subject, Audience and Issuer against the presented assertion. https://learn.microsoft.com/entra/workload-id/workload-identity-federation Trace ID: f1a78171-9306-4853-8799-06322d508d00 Correlation ID: e3146eff-d35a-46c3-be50-29962710dcec Timestamp: 2024-10-03 19:40:08Z

Error: Interactive authentication is needed. Please run:
az login

Error: Login failed with Error: The process '/usr/bin/az' failed with exit code 1. Double check if the 'auth-type' is correct. Refer to https://github.com/Azure/login#readme for more information.`

**Comment by kachawla** [comment]
> @kachawla First run failed with this issue, haven't had time to triage:
> 
> ```
> Error: AADSTS700213: No matching federated identity record found for presented assertion subject 'repo:radius-project/community:environment:issues'. Please note that the matching is done using a case-sensitive comparison. Check your federated identity credential Subject, Audience and Issuer against the presented assertion. https://learn.microsoft.com/entra/workload-id/workload-identity-federation Trace ID: f1a78171-9306-4853-8799-06322d508d00 Correlation ID: e3146eff-d35a-46c3-be50-29962710dcec Timestamp: 2024-10-03 19:40:08Z
> 
> Error: Interactive authentication is needed. Please run:
> az login
> 
> Error: Login failed with Error: The process '/usr/bin/az' failed with exit code 1. Double check if the 'auth-type' is correct. Refer to https://github.com/Azure/login#readme for more information.
> ```

Will sync up offline in triage. From what I can see, I might not have access to the federated identity that needs to be updated.

**Pull request merged**